api/src/main/resources/emails/dollarthresholdemail/content.html

```diff
@@ -29,7 +29,7 @@
 <body>
 <img src="${HEADER_IMG}">
 <div>Hello ${FIRST_NAME} ${LAST_NAME},</div>
-<div>This is a reminder that you have used ${USED_CREDITS} in free credits offered by the All of Us Program.
+<div>This is a reminder that your user ${USERNAME} has used ${USED_CREDITS} in free credits offered by the All of Us Program.
```

Comment:
wording nit: would "your account" be more appropriate here? In our system "user" is really an abstract representation of the person receiving the email, so from their perspective they might consider this to be their "account".

Reply:
makes sense - I'll check with the UX review crew

Reply:
done